common/changes/@microsoft/rush/feat-pnpm-patch_2022-07-27-03-39.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "rush-pnpm accepts patch and patch-commit",
+      "type": "none"
+    }
```

Comment:
The [rush-pnpm-patch-demo](https://github.com/chengcyber/rush-pnpm-patch-demo) README.md mentions:

> NOTE: patched dependency has the exactly same integrity of the raw dependency. It confuses rush build cache, so it's essential to invoke rebuild here to see the correct log output.

Could we fix this somehow? I don't think it should block this PR, but it seems like it could cause trouble and would be worth trying to fix.